packages/sdk/src/types/runtime/assets/material.ts

```diff
@@ -53,10 +61,26 @@ export class Material extends Asset implements MaterialLike {
         }
 
         this._color.copy(def.material.color);
+        observe(this._color, 'color', (...path: string[]) => this.materialChanged(path));

```

Comment:
Please put a comment above this line similar to the ones you'll find above `observe` calls in the Actor class. The patching system is still pretty cryptic. Explanatory comments will help future devs.